src/measurement_models.jl

```diff
@@ -0,0 +1,68 @@
+abstract type AbstractMeasurementModel end
+
+struct StandardODEMeasurementModel{IIP,F} <: AbstractMeasurementModel
+    f::F
+end
+StandardODEMeasurementModel(f::SciMLBase.AbstractODEFunction) = begin
+    f = SciMLBase.unwrapped_f(f)
```

Comment:
This isn't ideal, we lose all the cool new precompilation benefits here. Maybe it's fine for now? But it should be fixed at some point.

Reply:
do in some future PR